update-affiliated/update_astropy_helpers.py

```diff
@@ -111,21 +118,23 @@ def open_pull_request(fork, repo):
     run_command('git add astropy_helpers')
     run_command('git add ah_bootstrap.py')
     run_command('git add ez_setup.py')
-    run_command('git commit -m "Updated astropy-helpers to {0}"'.format(HELPERS_TAG))
+    run_command(
+        'git commit -m "Updated astropy-helpers to {0}"'.format(HELPERS_TAG))
 
     run_command('git push origin {0}'.format(BRANCH))
 
     print(tmpdir)
 
     report_user = '@astrofrog'
 
-    if username != 'astrofrog':
-        report_user = '@{} or @astrofrog'.format(username)
+    if user.login != 'astrofrog':
```

Comment:
Note: `user.login` is more flexible in case you want to use token instead of username+password in the future.